webchat-backend/src/main/resources/data.sql

```diff
@@ -0,0 +1,61 @@
+DROP TABLE IF EXISTS chat_message;
+DROP TABLE IF EXISTS chat;
+DROP TABLE IF EXISTS customer cascade;
+DROP TABLE IF EXISTS agent_role;
+DROP TABLE IF EXISTS agent cascade;
+DROP TABLE IF EXISTS role;
+
+CREATE TABLE customer (
+    customer_id INT PRIMARY KEY,
+    ip_address VARCHAR(250),
+    alias VARCHAR(250)
+);
+
+CREATE TABLE agent (
+    agent_id int PRIMARY KEY,
+    agent_name VARCHAR(250),
+    first_name VARCHAR(250),
```

Comment:
what's the difference between agent_name and first name

Reply:
That would be the username